web/db/migrations/20260724140000_iroh_rekey_bindings/migration.sql

```
@@ -0,0 +1,58 @@
+      g.initiator_binding_id IN (SELECT id FROM revoked)
+      OR g.acceptor_binding_id IN (SELECT id FROM revoked)
+    )
+  RETURNING g.id
+)
+INSERT INTO iroh_account_security_states (user_id, lan_discovery_generation, created_at, updated_at)
+SELECT DISTINCT user_id, 2, now(), now() FROM revoked
+ON CONFLICT (user_id) DO UPDATE
+  SET lan_discovery_generation = iroh_account_security_states.lan_discovery_generation + 1,
+      updated_at = now();
+--> statement-breakpoint
+DROP INDEX IF EXISTS "iroh_endpoint_bindings_active_app_instance_unique";
```

SQLFluff (4.2.2)

[error] DROP statement should use CONCURRENTLY to avoid locking the table.

(PG01)